core/python3Action/lib/prelauncher.py

```diff
@@ -97,6 +97,26 @@ def cannot_start(msg):
 except Exception as ex:
   cannot_start("Invalid action: %s\n" % str(ex))
 
+class Context:
+  def __init__(self, env):
+    self.function_name = env["__OW_ACTION_NAME"]
+    self.function_version = env["__OW_ACTION_VERSION"]
+    self.activation_id = env["__OW_ACTIVATION_ID"]
```

Comment:
Should we add `__OW_TRANSACTION_ID` as `request_id`? 🤔

Reply:
what's the value of providing both?
request_id isn't a usable user resource.

Reply:
We're surfacing the transaction id in errors and via a header in every response and as such it **might** be useful to have it available here as well for completeness sake. I guess we can always add it later as well though too.

Reply:
In fact, the header we respond with is `X-Request-Id`, containing the tid. As such, I decided to add request_id here as well, for completeness.